pom.xml

```diff
@@ -240,6 +240,11 @@
     <spark.test.home>${session.executionRootDirectory}</spark.test.home>
 
     <CodeCacheSize>1g</CodeCacheSize>
+
+    <!-- Thrift properties -->
+    <thrift.home>you-must-set-this-to-run-thrift</thrift.home>
+    <thrift.gen.dir>${basedir}/src/gen/thrift</thrift.gen.dir>
+    <thrift.args>-I ${thrift.home} --gen java:beans,hashcode,generated_annotations=undated</thrift.args>
```

Comment:
(just asking, I don't know maven): should this be set in global pom, or in thriftserver pom? How can this be used?

If it requires setting it manually in your pom, and the generated files are committed anyway, they I would not do it.

Reply:
why generated_annotations=undated? (it's not in readme.txt)

Reply:
> (just asking, I don't know maven): should this be set in global pom, or in thriftserver pom? How can this be used?
> 
> If it requires setting it manually in your pom, and the generated files are committed anyway, they I would not do it.

Current same as hive, make it like antlr4 is best but I may need some help... since I am particularly good at maven various plugin, I am learning the hive's approach.

Reply:
@juliuszsompolski Please see this PR: https://github.com/AngersZhuuuu/spark/pull/2